lib/hound/helpers/element.ex

```diff
@@ -201,6 +201,9 @@ defmodule Hound.Helpers.Element do
   You can also pass the selector as a tuple.
 
       element_displayed?({:name, "example"})
+      
+  Note: If you'd like to check presence of elements in the DOM use `element?/2`, 
+  `element_displayed?/1` will only consider elements that are always present in the DOM, either in visible or hidden state.
```

Comment:
I guess what's more important is that the documentation needs to reflect that there is an `element?/2` function present (albeit in a different module)

The intent and functionality of `element_displayed?` becomes clear after using it. 

A developer scouring through the documentation for `Hound.Helpers.Element` may think they can use `element_displayed?` when what they really wanted to use was `element?`

So, in light of this, do you think the second line that I wrote can just be discarded?